backend/pkg/api/types/validator_dashboard.go

```diff
@@ -269,6 +274,59 @@ type VDBTotalWithdrawalsData struct {
 
 type InternalGetValidatorDashboardTotalWithdrawalsResponse ApiDataResponse[VDBTotalWithdrawalsData]
 
+// ------------------------------------------------------------
+// Rocket Pool Tab
+type VDBRocketPoolTableRow struct {
+	Node      Address         `json:"node"`
+	StakedEth decimal.Decimal `json:"staked_eth"`
+	Minipools struct {
+		Total uint64 `json:"total"`
+		Leb16 uint64 `json:"leb_16"`
+		Leb8  uint64 `json:"leb_8"`
+	} `json:"minipools"`
+	Collateral    PercentageDetails[decimal.Decimal] `json:"collateral"`
+	AvgCommission float64                            `json:"avg_commission"`
+	Rpl           struct {
+		Claimed   decimal.Decimal `json:"claimed"`
+		Unclaimed decimal.Decimal `json:"unclaimed"`
+	} `json:"rpl"`
+	EffectiveRpl  decimal.Decimal `json:"effective_rpl"`
+	RplApr        float64         `json:"rpl_apr"`
+	SmoothingPool struct {
+		OptIn     bool            `json:"opt_in"`
```

Comment:
Frontend asked us to prefix bool variables with `Is` or `Has` from now on, so
```suggestion
		IsOptIn     bool            `json:"is_opt_in"`
```
or
```suggestion
		HasOptIn     bool            `json:"has_opt_in"`
```
depending on whether Opt-In denotes some state or possession.